src/components/Layout/__tests__/__snapshots__/layout.test.tsx.snap

```diff
@@ -135,6 +136,211 @@ exports[`Layout component renders correctly with data 1`] = `
                 </svg>
               </a>
             </li>
+            <li
+              class="nav__tabs"
+            >
+              <button
+                type="button"
+              >
+                <span
+                  class="sr-only"
+                >
+                  Locale
+                </span>
+                <svg
```

Comment:
This could be in the image folder.

Reply:
Since all the images in the Header file are svg tags, I think this should go into a seperate PR